crates/biome_fs/src/fs.rs

```diff
@@ -454,67 +453,65 @@ impl Display for FileSystemDiagnostic {
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub enum FsErrorKind {
     /// File not found
-    CantReadFile(String),
+    CantReadFile,
     /// Unknown file type
     UnknownFileType,
     /// Dereferenced (broken) symbolic link
-    DereferencedSymlink(String),
+    DereferencedSymlink,
     /// Too deeply nested symbolic link expansion
-    DeeplyNestedSymlinkExpansion(String),
+    DeeplyNestedSymlinkExpansion,
     /// Invalid UTF-8 characters in path.
-    NonUtf8Path(PathBuf),
+    NonUtf8Path,
```

Comment:
The `String` in these variants were the file path, but we already have this information in the `location(resource)` of the diagnostic itself. It was redundant information